pkg/vsphere/vm/vm_test.go

```diff
@@ -190,6 +191,13 @@ func TestVM(t *testing.T) {
 		t.Errorf("Failed to find fixed vm: %s", err)
 	}
 	assert.Equal(t, vm.Reference(), newVM.Reference())
+
+	// VM properties
+	var ovm mo.VirtualMachine
```

Comment:
@dougm the registervm methods are folder method or vApp method, how to inject the folder or vApp in simulator?